src/libopensc/card.c

```diff
@@ -1154,16 +1154,13 @@ sc_algorithm_info_t * sc_card_find_alg(sc_card_t *card,
 
```

Comment:
The indenting is broken here, isn't it?

Reply:
Sorry, how it is broken? Cannot see it.

Reply:
Oh, because I dropped one level of ifs, I see.